Hardcore.lua

```diff
@@ -33,6 +33,7 @@ Hardcore_Character = {
 	deaths = 0,
 	bubble_hearth_incidents = {},
 	played_time_gap_warnings = {},
+	trade_partners = {},
```

Comment:
Does this get added to their saved_variables if someone installs this new version?

Reply:
Not for existing characters that already have a saved variable file saved. We don't really support starting to use the addon for a existing run, but I guess it makes sense to support this case anyway, otherwise people who update to this version will get a lua error. I will push an update later today.

Reply:
Since this is a reacurring problem, I will handle it in a separate PR to solve it once and for all.